collectd/map.jinja

```diff
@@ -8,6 +8,7 @@
         'read_interval': 60,
         'file_logging': True,
         'remote_collector': False
+        'use_fqdn': True,
```

Comment:
I've asked the same question to @cznewt here, https://github.com/tcpcloud/salt-formula-grafana/pull/1#discussion_r84870374